app/test_main.py

```diff
@@ -1,3 +1,43 @@
+
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_human_age() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+    assert get_human_age(14, 14) == [0, 0]
+
+    assert get_human_age(15, 15) == [1, 1]
+
+    assert get_human_age(23, 23) == [1, 1]
+
+    assert get_human_age(24, 24) == [2, 2]
+
+    assert get_human_age(27, 27) == [2, 2]
```

Comment:
You can write just one test here because they all have the same concept: same input -> same output.